docs/references/beamfits_memo.tex

```diff
@@ -0,0 +1,154 @@
+of what these keywords mean, see the descriptions on pyuvdata's ReadTheDocs uvbeam\_parameters page. 
+The UVBeam parameter corresponding to each keyword is noted in parentheses.
+As with all FITS files, HISTORY and COMMENT cards are allowed.
+
+\begin{itemize}
+\item{\textbf{BTYPE}: should be set to either `power' or `efield' to designate a power or E-field beam type. (beam\_type)}
+\item{\textbf{NORMSTD}: should be set to one of `physical', `peak' or `solid\_angle'. (data\_normalization)}
+\item{\textbf{COORDSYS}: should be set to one of `az\_za', `sin\_zenith' or `healpix'. (pixel\_coordinate\_system)}
+\item{\textbf{TELESCOP}: Telescope name. (telescope\_name)}
+\item{\textbf{FEED}: Feed name (used to distinguish between different physical feed types on the same telescope). (feed\_name)}
+\item{\textbf{FEEDVER}: Feed version (used to distinguish between different versions of the same feed on the same telescope). (feed\_version)}
+\item{\textbf{MODEL}: Model name  (used to distinguish between different models of the same hardware). (model\_name)}
+\item{\textbf{MODELVER}: Model version (used to distinguish between different versions of the same models of the same hardware). (model\_version)}
+\item{\textbf{FEEDLIST}: Only for E-field beams. The list of feeds orientations represented in the data. (feed\_array)}
+\item{\textbf{NSIDE}: Only for HEALPix beams. The NSIDE of the HEALPix map. (nside)}
+\item{\textbf{ORDERING}: Only for HEALPix beams. The ordering parameter of the HEALPix map.(ordering)}
+\end{itemize}
+
+\subsection{Beam Data}
+The axes of the data array in the primary HDU depends on the pixelization scheme.
+
+\subsubsection{HEALPix beam axes}
+For HEALPix beams, the data axes are: 
+\begin{enumerate}
+\item{\textbf{PIX\_IND}: index into the HEALPix pixel array (in the HPX\_INDS extension) This is needed because partial sky maps are supported so pixels may be missing.}
+\item{\textbf{FREQ}: frequency}
+\item{\textbf{Polarization axis (STOKES or FEEDIND)}: STOKES for power beams give the polarization integer convention from uvfits (see AIPS memo 117), FEEDIND for E-field beams give the index into the FEEDLIST.}
+\item{\textbf{IF}: Spectral window number, currently only one spectral window is supported so this can only have length 1.}
+\item{\textbf{VECIND}: Basis vector index, the index into the dimensions of the basis vector array (stored in the BASISVEC extension). This is usually length 1 for power beams and length 2 for E-field beams.}
+\item{\textbf{COMPLEX}: real and imaginary (axis length 2)}
+\end{enumerate}
+
+\subsubsection{regularly pixelized beam axes}
+For regularly pixelized beams, the data axes are: 
+\begin{enumerate}
+\item{\textbf{First Image Axis (AZ or SIN\_X)}: Coordinates for the first image axis. The axis name is dictated by the COORDSYS keyword.}
+\item{\textbf{Second Image Axis (ZA or SIN\_Y)}: Coordinates for the second image axis. The axis name is dictated by the COORDSYS keyword.}
```

Comment:
Given the similarity between "AZ" and "ZA", it might be helpful to spell out Azimuth and Zenith Angle somewhere.

Reply:
Also, I think you need to have map projection parameters in here somewhere for the regularly gridded case, or at least the pixel scale.

Reply:
The pixel scale is in (i.e. the corresponding CDELTn keyword is set). I'm not super familiar with the map projection parameters, I can see needing them for the sin_zenith projection but do you need them for the azimuth, zenith angle case?

Reply:
I think I misunderstood what this document would look like when finished. I see `CDELT` in `beamfits.py` but not in this file. Will all of those parameters be automatically imported?

Reply:
I just went off and read some WCS stuff and I'm still not totally sure I know what to add, beyond the CDELT keywords. I'm still making my way through the 3 papers, but do you have some specific keywords in mind?

Reply:
I added some language to the tex about all the axes having the standard `CRVALn', `CRPIXn', and `CDELTn' keywords and sometimes having the `CUNITn' keyword for units. I consider that those are required to specify any FITs axis, so I wasn't spelling them out here, I was just explaining what the axes were. Is that too opaque?

Reply:
I also just realized that the `CUNITn` keyword wasn't being set for the image axes so I added that to the code.